odoo/cli/command.py

```diff
@@ -18,31 +18,13 @@ def __init_subclass__(cls):
         cls.name = cls.name or cls.__name__.lower()
         commands[cls.name] = cls
 
+    @classmethod
+    def prog_name(self):
+        return Path(sys.argv[0]).name
```

Comment:
```suggestion
    prog_name = Path(sys.argv[0]).name
```
many subcommand could benefit from using the attribute